pytorch_lightning/trainer/trainer.py

```diff
@@ -438,6 +445,7 @@ def __init__(
             track_grad_norm,
             accumulate_grad_batches,
             terminate_on_nan,
```

Comment:
internally we shall update codebase to use the new arg only

Reply:
Could you please clarify?

Reply:
Likely meant the same as in https://github.com/PyTorchLightning/pytorch-lightning/pull/9175/files#r699125191